commands/core/cli.drush.inc

```diff
@@ -70,14 +70,98 @@ function drush_cli_core_cli() {
 function _drush_core_cli_get_commands() {
   $commands = drush_get_commands();
   $ignored_commands = ['help', 'drush-psysh', 'php-eval', 'core-cli', 'php'];
+  $php_keywords = _drush_core_cli_get_php_keywords();
 
   foreach ($commands as $name => $config) {
-    // Ignore some commands that don't make sense inside PsySH, are hidden, or
-    // are aliases.
-    if (in_array($name, $ignored_commands) || !empty($config['hidden']) || ($name !== $config['command'])) {
+    // Ignore some commands that don't make sense inside PsySH, are PHP keywords
+    // are hidden, or are aliases.
+    if (in_array($name, $ignored_commands) || in_array($name, $php_keywords) || !empty($config['hidden']) || ($name !== $config['command'])) {
```

Comment:
@weitzman this just removes the command if it's conflicts, we could potentially prefix it with drush or something? This is very edge-casey, not sure if it's worth doing that or not?